brayns/parameters/ApplicationParameters.h

```diff
@@ -126,6 +133,7 @@ class ApplicationParameters : public AbstractParameters
     std::string _streamId;
     std::string _tmpFolder;
     bool _synchronousMode{false};
+    size_t _imageStreamFPS{60};
```

Comment:
we are become ambitious now :-)

Reply:
It's just an upper limit. We will never render that fast anyways :)